src/store/flow/flow.ts

```diff
@@ -171,7 +171,15 @@ export const actions: ActionTree<IFlowsState, IRootState> = {
   async flow_persist({ state, getters, commit }, { persistRoute, flowContainer }): Promise<IContext> {
     const restVerb = flowContainer.created ? 'put' : 'post'
     const oldCreatedState = flowContainer.created
-    return axios[restVerb](persistRoute, omit(flowContainer, ['created']))
+
+    const flowContainerCleanedResources = cloneDeep(flowContainer)
```

Comment:
To separate concerns of interrogation and mutation, we could potentially do without the heavier `cloneDeep()`, favouring collecting a list of resources in this first pass, then afterward rewrite the same `resources` property, maybe something like:
```
flowContainer.resources = without(flowContainer.resources, orphanedResources)
```